CycloneDX.Core.Tests/ComponentServiceTests.cs

```diff
@@ -43,7 +42,7 @@ public async Task RecursivelyGetComponents_ReturnsComponents()
             {
                 new NugetPackage { Name = "Package1", Version = "1.0.0" },
                 new NugetPackage { Name = "Package2", Version = "1.0.0" },
-                new NugetPackage { Name = "Package3", Version = "1.0.0" },
+                new NugetPackage { Name = "Package3", Version = "1.0.0" }

```

Comment:
There's no value in a change like this. And by having trailing commas it reduces noise if an additional element is added in the future.

Reply:
Yeah, right now as there is not element, the commo look like redundant, so I removed that!